sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/resourcemanager/AzureEventHubsResourceManagerAutoConfiguration.java

```diff
@@ -42,7 +43,11 @@ public class AzureEventHubsResourceManagerAutoConfiguration extends AzureService
     @Bean
     @ConditionalOnMissingBean
     @ConditionalOnProperty(prefix = AzureEventHubsProperties.PREFIX, value = "namespace")
-    @ConditionalOnMissingProperty(prefix = AzureEventHubsProperties.PREFIX, value = "connection-string")
+    @ConditionalOnMissingProperty({
+        AzureEventHubsProperties.PREFIX + ".connection-string",
+        AzureGlobalProperties.PREFIX + ".credential.token-credential-bean-name",
+        AzureEventHubsProperties.PREFIX + ".credential.token-credential-bean-name"
+    })
```

Comment:
But the credential can come from all the other spring.cloud.azure.credential.xxx properties

Reply:
Yes, not covered all the credentials, we should add all credential types here, right?

Reply:
Will be improved later.